.github/workflows/ci.yml

```diff
@@ -38,35 +34,32 @@ jobs:
             git clone https://github.com/LLNL/libROM.git
             cd libROM
             mkdir build
-            export CC=mpicc
-            export CXX=mpicxx
-            scripts/setup.sh
             cd build
-            cmake -DCMAKE_BUILD_TYPE=Debug -DUSE_MFEM=On ..
+            cmake .. -DCMAKE_TOOLCHAIN_FILE=${TOOLCHAIN_FILE} -DCMAKE_BUILD_TYPE=Debug -DUSE_MFEM=${USE_MFEM} -DMFEM_USE_GSLIB=${MFEM_USE_GSLIB}
             make
-            cmake -DCMAKE_BUILD_TYPE=Optimized -DUSE_MFEM=On ..
+            cmake .. -DCMAKE_TOOLCHAIN_FILE=${TOOLCHAIN_FILE} -DCMAKE_BUILD_TYPE=Optimized -DUSE_MFEM=${USE_MFEM} -DMFEM_USE_GSLIB=${MFEM_USE_GSLIB}
             make
       - uses: ./.github/workflows/run_tests
-  mac:
-    runs-on: macos-latest
-    steps:
-      - name: Install Mac dependencies
-        run: |
-            brew install open-mpi
-            brew install openblas
-            brew install lapack
-            brew install scalapack
-            brew install hdf5
-      - name: Check out libROM
-        uses: actions/checkout@v2
-      - uses: ./.github/workflows/checkout_repo
-      - name: Build libROM
-        run: |
-            export FC=/usr/local/bin/gfortran-10
-            mkdir ${GITHUB_WORKSPACE}/build
-            cd ${GITHUB_WORKSPACE}/build
-            cmake -DCMAKE_BUILD_TYPE=Debug ..
-            make
-            cmake -DCMAKE_BUILD_TYPE=Optimized ..
-            make
-      - uses: ./.github/workflows/run_tests
\ No newline at end of file
+  # mac:
+  #   runs-on: macos-latest
+  #   steps:
+  #     - name: Install Mac dependencies
+  #       run: |
+  #           brew install open-mpi
+  #           brew install openblas
+  #           brew install lapack
+  #           brew install scalapack
+  #           brew install hdf5
+  #     - name: Check out libROM
+  #       uses: actions/checkout@v2
+  #     - uses: ./.github/workflows/checkout_repo
+  #     - name: Build libROM
+  #       run: |
+  #           export FC=/usr/local/bin/gfortran-10
+  #           mkdir ${GITHUB_WORKSPACE}/build
+  #           cd ${GITHUB_WORKSPACE}/build
+  #           cmake -DCMAKE_BUILD_TYPE=Debug ..
+  #           make
+  #           cmake -DCMAKE_BUILD_TYPE=Optimized ..
+  #           make
+  #     - uses: ./.github/workflows/run_tests
```

Comment:
@siuwuncheung , we know the issue and the mac workflow is separate from the linux one so test is still effective. But since this is a public repo which many other people use, for visibility to them, I **personally** think it would be better to comment this out, until the mac ci workflow is actually established. But we can uncomment this if we think the conflict is more of the issue.

Reply:
Sounds good to me. I'll approve the PR right away.